source/math/FloatPoint2DVectorList.ooc

```diff
@@ -87,10 +87,52 @@ FloatPoint2DVectorList: class extends VectorList<FloatPoint2D> {
 		}
 		result
 	}
+	_swap: static func (array: FloatPoint2D*, i, j: Int) {
```

Comment:
Hmm, might be nice with a generic `swap` in `VectorList`. Not for this PR though.